### PR TITLE
-B and -C editorial review edits

### PR DESCRIPTION
**Pull request by jimfenton**

### CODE REVIEW

File: sp800-63b/sec5_authenticators.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Verifiers of look-up secrets SHALL prompt the claimant for the next secret from
...
 
-Look-up secrets SHALL be generated using an approved random bit generator [[SP 800-90Ar1]](#SP800-90Ar1) and SHALL have at least 20 bits of entropy. When look-up secrets have less than 64 bits of entropy, the verifier SHALL implement a rate-limiting mechanism that effectively limits the number of failed authentication attempts that can be made on the subscriber's account as described in [Section 5.2.2](#throttle).
+For look-up secrets have less than 64 bits of entropy, the verifier SHALL implement a rate-limiting mechanism that effectively limits the number of failed authentication attempts that can be made on the subscriber's account as described in [Section 5.2.2](#throttle).
```
Comment (jricher) [inline sp800-63b/sec5_authenticators.md]: Missing "that":

> For look-up secrets that have less than...

File: sp800-63b/sec7_session.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Secrets used for session binding:
 2. SHALL be generated by an approved random bit generator [[SP 800-90Ar1]](#SP800-90Ar1) and contain at least 64 bits of entropy.
 3. SHALL be erased or invalidated by the session subject when the subscriber logs out.
 4. SHOULD be erased on the subscriber endpoint when the user logs out or when the secret is deemed to have expired.
-5. SHOULD not be placed in insecure locations such as HTML5 Local Storage due to the potential exposure of local storage to XSS attacks.
+5. SHOULD not be placed in insecure locations such as HTML5 Local Storage due to the potential exposure of local storage to cross-site scripting (XSS) attacks.
```
Comment (jricher) [inline sp800-63b/sec7_session.md]: normative SHOULD NOT? (or is our normative language only "SHOULD" without the negative?)

File: sp800-63c/cover.md [hunk 3]
```diff
@@ -154,7 +154,7 @@ The Information Technology Laboratory (ITL) at the National Institute of Standar
...
 
-These guidelines provide technical requirements for federal agencies implementing digital identity services and are not intended to constrain the development or use of standards outside of this purpose. This guideline focuses on the use of use of federated identity, and the use of assertions to implement identity federations. Federation allows a given credential service provider to provide authentication and (optionally) subscriber attributes to a number of separately administered relying parties. Similarly, relying parties may use more than one credential service provider.
+These guidelines provide technical requirements for federal agencies implementing digital identity services and are not intended to constrain the development or use of standards outside of this purpose. This guideline focuses on the use of federated identity, and the use of assertions to implement identity federations. Federation allows a given credential service provider to provide authentication and (optionally) subscriber attributes to a number of separately-administered relying parties. Similarly, relying parties may use more than one credential service provider.
```
Comment (jricher) [inline sp800-63c/cover.md]: Remove extraneous comma here: 

> use of federated identity and the use of assertions

File: sp800-63c/sec10_usability.md [hunk 4]
```diff
@@ -8,7 +8,7 @@
...
 
-From the usability perspective, one of the major potential benefits of federated identity systems is to address the problem of user fatigue associated with managing multiple authenticators. While this has historically been a problem with usernames and passwords, as the market evolves, users having to manage many authenticators – whether physical or digital – presents a usability challenge.
+From the usability perspective, one of the major potential benefits of federated identity systems is to address the problem of user fatigue associated with managing multiple authenticators. While this has historically been a problem with usernames and passwords, the increasing need for users to manage many authenticators – whether physical or digital – presents a usability challenge.
```
Comment (jricher) [inline sp800-63c/sec10_usability.md]: Dashes should not have spaces around them according to style guides.

File: sp800-63c/sec10_usability.md [hunk 5]
```diff
@@ -85,12 +84,11 @@ Many properties of identity have implications for how users manage identities, b
...
+Many factors can influence user adoption of federated identity systems. As with any technology, users may value some factors more than others. Users often weigh perceived benefits versus risks before making technology adoption decisions. It is critical that IdPs and RPs provide users with sufficient information to enable them to make informed decisions. The concepts of trust and tiers of trust — fundamental principles in federated identity systems — can drive user adoption. Finally, a positive user experience may also result in increased user demand for federation, triggering increased adoption by RPs.
```
Comment (jricher) [inline sp800-63c/sec10_usability.md:87]: Dashes should not have spaces around them as per style guides.

File: sp800-63c/sec6_assertions.md [hunk 6]
```diff
@@ -80,17 +80,17 @@ The assertion signature SHALL either be a digital signature using asymmetric key
...
 
 All encryption of assertions SHALL use approved cryptography.
 
-When assertions are passed through third parties, such as a browser, the actual assertion SHALL be encrypted. For example, the XML SAML assertion using XML-Encryption. For assertions that are passed directly between IDP and RP, the actual assertion MAY be encrypted. If it is not, the assertion SHALL be sent over an authenticated protected channel.
+When assertions are passed through third parties, such as a browser, the actual assertion SHALL be encrypted. This encryption MAY include, for example, the XML SAML assertion using XML-Encryption. For assertions that are passed directly between IDP and RP, the actual assertion MAY be encrypted. If it is not, the assertion SHALL be sent over an authenticated protected channel.
```
Comment (jricher) [inline sp800-63c/sec6_assertions.md]: This is an example and should not have a normative statement attached to it in this way. Suggest rewording the sentence as:

> For example, a SAML assertion can be encrypted using XML-Encryption, or an OpenID Connect ID Token can be encrypted using JSON Web Encryption (JWE).

Not sure how I missed that we didn't have an OIDC example here, fwiw.

File: sp800-63c/sec6_assertions.md [hunk 7]
```diff
@@ -80,17 +80,17 @@ The assertion signature SHALL either be a digital signature using asymmetric key
...
 
 > Note: Assertion encryption is required at FAL2 and FAL3.
 
 #### 6.2.4 Audience Restriction
 
-Assertions SHALL use audience restriction techniques to allow an RP to recognize whether or not it is the intended target of an issued assertion. All RPs SHALL check that the audience of an assertion contains an identifer for their RP to prevent the injection and replay of an assertion generated for one RP at another RP.
```
Comment (jricher) [inline sp800-63c/sec6_assertions.md:93]: 1qqtlj

File: sp800-63c/sec10_usability.md (hunk 5 above)
Comment (jricher) [reply sp800-63c/sec10_usability.md:87]: ALSO: we should be consistent with the use of emdashes vs. endashes throughout. Emphasized parenthetical phrases like this should use the (wider) emdash.

http://www.thepunctuationguide.com/em-dash.html

Endashes are used for things like ranges of numbers.

**Review by jricher — CHANGES_REQUESTED** [review]
Overall good, just a couple of minor tweaks.

File: sp800-63c/sec6_assertions.md (hunk 7 above)
Comment (jricher) [reply sp800-63c/sec6_assertions.md:93]: In case it's unclear, this edit is just fine. I just can't believe this typo lasted this long. :)

File: sp800-63b/sec5_authenticators.md (hunk 1 above)
Comment (jimfenton) [reply sp800-63b/sec5_authenticators.md]: good catch, thanks.

File: sp800-63b/sec7_session.md (hunk 2 above)
Comment (jimfenton) [reply sp800-63b/sec7_session.md]: Also thanks.

File: sp800-63c/cover.md (hunk 3 above)
Comment (jimfenton) [reply sp800-63c/cover.md]: Agreed

File: sp800-63c/sec10_usability.md (hunk 4 above)
Comment (jimfenton) [reply sp800-63c/sec10_usability.md]: Style guides I have seen disagree on this, and elsewhere our copy editors have put spaces around them so we should do the same here.

File: sp800-63c/sec10_usability.md (hunk 5 above)
Comment (jimfenton) [reply sp800-63c/sec10_usability.md:87]: I thought I had caught all the en dashes but I hadn't. Several of them in Section 6 seem to be setting off a definition from a term, which I will replace with a colon (without the leading space, obviously).

File: sp800-63c/sec6_assertions.md (hunk 6 above)
Comment (jimfenton) [reply sp800-63c/sec6_assertions.md]: Done, reads better.

File: sp800-63c/sec6_assertions.md (hunk 7 above)
Comment (jimfenton) [reply sp800-63c/sec6_assertions.md:93]: 😄  I can't believe how long some of the typos lived, especially the misspellings of "assessments."

**Review by jricher — APPROVED** [review]
